### PR TITLE
fix: Full Screen in Landscape mode cut the video #621

### PR DESCRIPTION
**Pull request by longtn-imt**
fix: Full Screen in Landscape mode cut the video #621

### CODE REVIEW

**Comment by longtn-imt** [comment]
Screenshot_1720602565

**Comment by Raghavan2005** [comment]
nice can you upload this on pub.dev

**Comment by ahmedelameer34** [comment]
Is it need any permissions or keys from YouTube or videos owners?

File: packages/youtube_player_flutter/lib/src/player/youtube_player.dart [hunk 1]
```diff
@@ -297,8 +297,9 @@ class _YoutubePlayerState extends State<YoutubePlayer> {
         children: [
           Transform.scale(
             scale: controller.value.isFullScreen
-                ? (1 / _aspectRatio * MediaQuery.of(context).size.width) /
-                    MediaQuery.of(context).size.height
+                ? 1
```
Comment (memoriasIT) [inline packages/youtube_player_flutter/lib/src/player/youtube_player.dart:300]: `isFullScreen ? 1 : 1,`

Doesn't make much sense to me, better just use 1 directly 👍

**Review by sarbagyastha — APPROVED** [review]

**Pull request merged**